src/plugins/Sharesite/webui/EditToadlet.java

```diff
@@ -114,62 +126,76 @@ private void addNodes( HTMLNode form, String name, String desc, String text, Str
 		vals = new String[] { "textInput", "20", "80", "font-size: medium;" };
 		textDiv.addChild("textarea", attrs, vals, text);
 
+		HTMLNode insertHour = editBox.content.addChild("p");
```

Comment:
that’s true … it’s been too long since I’ve seen type-prefixes. And there is no reason to disambiguate this here (which is what the surrounding code does with things like aUri), so I can change this without making the style inconsistent. Thank you!